java/configuration/src/main/java/sleeper/configuration/properties/instance/EMRServerlessProperty.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.configuration.properties.instance;
+
+import sleeper.configuration.properties.SleeperPropertyIndex;
+
+import java.util.List;
+
+public interface EMRServerlessProperty {
+
+    UserDefinedInstanceProperty BULK_IMPORT_EMR_SERVERLESS_CLASS_NAME = Index
+            .propertyBuilder("sleeper.bulk.import.emr.serverless.class.name")
+            .description("The class to use to perform the bulk import.")
+            .propertyGroup(InstancePropertyGroup.BULK_IMPORT)
+            .defaultValue(
+                    "sleeper.bulkimport.job.runner.dataframelocalsort.BulkImportDataframeLocalSortDriver")
+            .runCDKDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty BULK_IMPORT_EMR_SERVERLESS_ARCHITECTURE = Index
+            .propertyBuilder("sleeper.bulk.import.emr.serverless.architecture")
+            .description("The architecture for EMR Serverless to use. X86_64 or ARM (Coming soon)")
+            .propertyGroup(InstancePropertyGroup.BULK_IMPORT).defaultValue("X86_64")
+            .runCDKDeployWhenChanged(true).build();
```

Comment:
This property could do with a validation predicate to ensure that only X86_64 is allowed at the moment.

Reply:
Added